Formula/babel.rb

```diff
@@ -4,8 +4,8 @@
 class Babel < Formula
   desc "Compiler for writing next generation JavaScript"
   homepage "https://babeljs.io/"
-  url "https://registry.npmjs.org/@babel/cli/-/cli-7.4.4.tgz"
-  sha256 "aa6b4c8d5564e44f68e0ac1198e8a13ae641fcd9af3119e38538e34a691f1dbb"
+  url "https://github.com/babel/babel/archive/v7.4.5.tar.gz"
+  sha256 "0e046a7b2ba5207b741bfb8de34dd4a15d6d4efa4deee0f8a1a7e4a0a4a8f045"
```

Comment:
But you should still use the `@babel/core` npm package here instead of the github monorepo, which contains all the `babel` related packages in one repository plus some tooling.

Reply:
I see!